blockstack_client/backend/drivers/ipfs.py

```diff
@@ -190,7 +180,9 @@ def storage_init(conf, **kwargs):
                 IPFS_PORT = parser.get('ipfs', 'port')
             
             if parser.has_option('ipfs', 'compress'):
-                AWS_COMPRESS = (parser.get('ipfs', 'compress', 'false').lower() in ['true', '1'])
+                IPFS_COMPRESS = (parser.get('ipfs', 'compress', 'false').lower() in ['true', '1'])
+
+   ipfs_api = ipfsapi.connect( IPFS_SERVER, IPFS_PORT )
```

Comment:
Does the object returned by `ipfsapi.connect()` automatically try to re-connect if e.g. the IPFS daemon restarts?  I'd imagine that this could happen while Blockstack is running in the background (e.g. the user upgrades IPFS).